icons/balloon.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "peteruithoven"
+  ],
+  "tags": [
+    "party, festival, congratulations"
+  ],
```

Comment:
Some more tags thanks to Chatgpt
```suggestion
    "celebration",
    "decoration",
    "colorful",
    "floating",
    "fun",
    "birthday",
    "event",
    "entertainment",
  ],
```

Reply:
List is vertical now and I've added those tags. 
This also made me realize they were added as one string. I'll try to find a relevant repo to open an suggestion for validation.